ui/shared/components/panel/variants/VariantGene.test.js

```diff
@@ -4,12 +4,12 @@ import Adapter from 'enzyme-adapter-react-16'
 import configureStore from 'redux-mock-store'
 import VariantGene from './VariantGene'
```

Comment:
A little confusing for the names of VariantGene and VariantGenes. This one must be VariantGenes class, right?